app/models/reports/repository.rb

```diff
@@ -120,21 +133,45 @@ def reporting_schema_v2?
       end
     end
 
+    private def registered_patients_query_v2(region)
+      Reports::FacilityState.for_facility(region).order(:month_date).pluck(:month_date, :monthly_registrations)
```

Comment:
I feel using a `where` directly is more transparent, and familiar. Wdyt? Oh and we drop the `Reports::` prefix since we're in the same module, ya?

```ruby
FacilityState
  .where(facility_region_id: region.id)
  .order(:month_date)
  .pluck(:month_date, :monthly_registrations)
```